lib/src/main/java/uk/gov/di/ipv/cri/passport/library/domain/VerifiableCredential.java

```diff
@@ -0,0 +1,69 @@
+package uk.gov.di.ipv.cri.passport.library.domain;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import uk.gov.di.ipv.cri.passport.library.persistence.item.PassportCheckDao;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class VerifiableCredential {
+
+    @JsonProperty private String resourceId;
+    @JsonProperty private CredentialSubject credentialSubject;
+    @JsonProperty private Evidence evidence;
```

Comment:
Something I'm curious about is what this but of a VC looks like for something that isn't evidence based. I.e i believe different cri's have different gpg45 score structures essentially.

Might be something to double check with george and will help us decide a good naming convention across all the CRI's